core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
@hachikuji A quick follow-up question: Is it possible for the current API to return an offset fetch response with various error codes associated with the topics? I'm trying to think if we can remove the internal "error_code" from the schema. Thanks.